Packs/EWS/Integrations/EWSO365/EWSO365.py

```diff
@@ -2384,18 +2383,17 @@ def sub_main():
     is_test_module = False
     params = demisto.params()
     args = prepare_args(demisto.args())
+    command = demisto.command()
     # client's default_target_mailbox is the authorization source for the instance
-    params['default_target_mailbox'] = args.get('target_mailbox',
-                                                args.get('source_mailbox', params['default_target_mailbox']))
-
+    params['default_target_mailbox'] = args.get('target_mailbox', args.get('source_mailbox', params['default_target_mailbox']))
+    if params.get('upn_mailbox') and not(args.get('target_mailbox')):
+        params['default_target_mailbox'] = params.get('upn_mailbox')
     try:
         client = EWSClient(**params)
         start_logging()
-
         # replace sensitive access_token value in logs
         add_sensitive_log_strs(client.credentials.access_token.get('access_token', ''))
 
-        command = demisto.command()
```

Comment:
why did you remove this?

Reply:
I moved it next to the args but since I don't need it I'll return it back